libraries/AP_HAL/board/linux.h

```diff
@@ -227,6 +227,7 @@
 #define HAL_BARO_DEFAULT HAL_BARO_MS5611_SPI
 #define HAL_BARO_MS5611_NAME "ms5611"
 #define HAL_COMPASS_DEFAULT HAL_COMPASS_BBBMINI
+#define HAL_AIRSPEED_MS4515DO_I2C_BUS 2
```

Comment:
Hi @OXINARF I'm double about that.
On AP_Airspeed_MS4525.cpp is checking the definetion for HAL_AIRSPEED_MS4515DO_I2C_BUS, in negative case a value of 1 is set, but in case of BBBMini would be the wrong bus - at same time I checked at GCS(qgroundcontrol) and there is reference to bus i2c 0 and 1 too - but not 2, so I couldn't figure out if HAL_AIRSPEED_MS4515DO_I2C_BUS would be set by gcs or if it's really necessary to avoid be set 1 in case of undefined value for HAL_AIRSPEED_MS4515DO_I2C_BUS.